gef.py

```diff
@@ -4822,6 +4822,8 @@ def do_invoke(self, argv, *args, **kwargs):
         elif args.lang == "asm":
             asm_type = self.format_matrix[args.bitlen][2]
             out = "buf {0} {1}".format(asm_type, sdata)
+        elif args.lang == "hex":
+            out = "".join(f'{x:02x}' for x in data)
```

Comment:
We support (for now) Python < 3.6 where f-strings don't exist. Please use the `.format` notation.

```suggestion
            out = "".join('{:02x}'.format(x) for x in data)
```